app/assets/javascripts/location_search.js

```diff
@@ -11,13 +11,17 @@ window.ST = window.ST || {};
     var autocomplete = new window.google.maps.places.Autocomplete(searchInput);
     autocomplete.setTypes(['geocode']);
 
+    boundingboxInput.value = null;
+
     window.google.maps.event.addListener(autocomplete, 'place_changed', function(){
       var place = autocomplete.getPlace();
       if(place != null) {
         if(place.geometry != null) {
           coordinateInput.value = place.geometry.location.toUrlValue();
           statusInput.value = window.google.maps.places.PlacesServiceStatus.OK;
-          boundingboxInput.value = place.geometry.viewport.toUrlValue();
+          if (place.geometry.viewport) {
+            boundingboxInput.value = place.geometry.viewport.toUrlValue();
```

Comment:
I guess, we need this if statement to the other "place.geometry.viewport.toUrlValue();" too